dev-docs/best-practices/microservices.md

```diff
@@ -144,3 +144,14 @@ type Hook {
   status(hookGroupId: ID = hookGroupId, hookId: ID = hookId): HookStatus
 }
 ```
+
+## Scopes
+
+Any scopes that the service needs in its credentials should be listed in `service/<name>/scopes.yml`.
+The exception is scopes for Azure table access.
+
+## Azure Tables
+
+Any azure tables or containers that a service uses in production should be described in `service/<name>/azure.yml`, under keys `containers` and `tables`.
+The `yarn generate` logic will ensure that appropriate scopes are added to the service's static clients.
```

Comment:
I wonder if we should even take the next step and automatically inject azure table creation into lib-loader based on this :bulb:

Reply:
Given the desire to move to postgres, I didn't want to over-complicate things here.